SIGSEGV: segmentation violation with 1.51.0

### TL;DR

Got at a hardened stable Gentoo Linux today with 1.51.0:
```
 ./bin/create-server.sh hi-t-intel-ltsrc-bp-cl

 >>> using Hetzner project "test"
 creating 1 system/s: hi-t-intel-ltsrc...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x55e808280c2d]

goroutine 1 [running]:
github.com/hetznercloud/cli/internal/cmd/server.init.func13({0x55e80860e820, 0xc000224580}, 0xc0002f9b08, {0x55e80855bfe0?, 0xc0000104b0?})
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/internal/cmd/server/create.go:150 +0x6d
github.com/hetznercloud/cli/internal/cmd/base.(*CreateCmd).CobraCommand.func1(0xc0002f9b08, {0xc00020fe00, 0x0, 0x8})
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/internal/cmd/base/create.go:74 +0x228
github.com/spf13/cobra.(*Command).execute(0xc0002f9b08, {0xc000000220, 0x8, 0x8})
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/vendor/github.com/spf13/cobra/command.go:1015 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002c0308)
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/vendor/github.com/spf13/cobra/command.go:1148 +0x46f
github.com/spf13/cobra.(*Command).Execute(0x55e80860e820?)
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/vendor/github.com/spf13/cobra/command.go:1071 +0x13
main.main()
        /var/tmp/portage/app-admin/hcloud-1.51.0/work/hcloud-1.51.0/cmd/hcloud/main.go:31 +0x11a


```


### Expected behavior

return code 0

### Observed behavior

:-o

### Minimal working example

_No response_

### Log output

```plain text

```

### Additional information

_No response_

--- a/internal/cmd/loadbalancer/create.go
+++ b/internal/cmd/loadbalancer/create.go
@@ -103,6 +103,9 @@ var CreateCmd = base.CreateCmd{
 		if err != nil {
 			return nil, nil, err
 		}
+		if loadBalancer == nil {
+			return nil, nil, fmt.Errorf("Load Balancer not found: %d", result.LoadBalancer.ID)
+		}
 		cmd.Printf("Load Balancer %d created\n", loadBalancer.ID)
 
 		if err := changeProtection(s, cmd, loadBalancer, true, protectionOpts); err != nil {

--- a/internal/cmd/server/create.go
+++ b/internal/cmd/server/create.go
@@ -118,6 +118,9 @@ var CreateCmd = base.CreateCmd{
 		if err != nil {
 			return nil, nil, err
 		}
+		if server == nil {
+			return nil, nil, fmt.Errorf("server not found: %d", result.Server.ID)
+		}
 
 		cmd.Printf("Server %d created\n", result.Server.ID)
 

